fmom/pxpypze.go

```diff
@@ -211,3 +211,17 @@ func (p4 *PxPyPzE) Set(p P4) {
 	p4[2] = p.Pz()
 	p4[3] = p.E()
 }
+
+func dotProduct(p1, p2 *PxPyPzE)float64{
+	dotproduct := p1.Px()*p2.Px()+p1.Py()*p2.Py()+p1.Pz()*p2.Pz()
+	return dotproduct
+}
+
+//return the cosine of the angle between the momentum of two jets
```

Comment:
Please use the canonical way of documenting Go code:

```go
// CosTheta returns the cosine of the angle between the momentum of two 4-vectors.
func CosTheta(...) ...
```

please also specify whether the returned angle is in radians or in degrees, and the expected range ([0,2pi), [-pi,+pi), ...)